src/main/osd/osd.h

```diff
@@ -141,6 +141,9 @@ typedef enum {
     OSD_PID_PROFILE_NAME,
     OSD_PROFILE_NAME,
     OSD_RSSI_DBM_VALUE,
+    #ifdef USE_RX_SNR_DBM
```

Comment:
Should go at the end, and not be conditional, to yield constant and unchangeable values.

Reply:
Will do.
I'll also check redo the other enums in this file.